action.yml

```
@@ -14,6 +14,10 @@ inputs:
     description: Force install CML if it exists.
     default: false
     required: false
+  ci:
+    description: Run `cml ci` after installing CML.
```

```suggestion
    description: Also run `cml ci` (`true`) or flags to forward to `cml ci` (str).
```

I think? i.e. `ci: '--user-email=${{ github.actor }}@users.noreply.github.com'`